### DISCUSSION TITLE
About a Unit used by BGRAFlashProgressBar

### DISCUSSION THREAD

**Discussion by maxm74** [bgrabitmap/bgracontrols#209] [General]
I am working on the ProgressBar. I saw that a class derived from TPersistent in the bgradrawerflashprogressbar.pas unit is used to do the drawing.
Every property of the TBGRAFlashProgressBar class sets/reads the equivalent property in TBGRADrawerFlashProgressBar, if I have to do Invalidate I have to call the OnChange event. This doesn't make much sense if the TBGRADrawerFlashProgressBar class is not used anywhere else. Can I delete this class and move the content to the main TBGRAFlashProgressBar class?
Thanks.

**Comment by lainz** [comment]
Sure change what you need-
